credentials/src/main/kotlin/web5/sdk/credentials/VerifiableCredential.kt

```diff
@@ -134,13 +129,14 @@ public class VerifiableCredential internal constructor(public val vcDataModel: V
         false -> throw IllegalArgumentException("expected data to be parseable into a JSON object")
       }
 
-      val credentialSubject = CredentialSubject.builder()
+      val credentialSubject = CredentialSubject.Builder()
```

Comment:
you mentioned we could possibly do without using the builder pattern - could you show me a snippet of what that might look like?

Reply:
Yea! I went ahead and got away from builder now, we are more pure kotlin now

Pretty good article here on why builder is outdated and in kotlin can make a builder with data classes with like 10% of the code:
https://www.baeldung.com/kotlin/builder-pattern